roles/dosomething.base/tasks/install-common-tools.yml

```diff
@@ -12,3 +12,8 @@
     - screen
     - bind9utils
     - sysstat
+
+- name: Disable AppArmor
+  sudo: yes
+  apt: name=apparmor state=absent
+  notify: restart server
```

Comment:
We can to run the rest of the steps, I know what you're talking about...
I'll add it and re-push the commit.

On Mon, May 11, 2015 at 1:48 PM, Sergii Tkachenko <notifications@github.com>
wrote:

> In roles/dosomething.base/tasks/install-common-tools.yml
> <https://github.com/DoSomething/tower/pull/9#discussion_r30063233>:
>
> > @@ -12,3 +12,8 @@
> >      - screen
> >      - bind9utils
> >      - sysstat
> > +
> > +- name: Disable AppArmor
> > +  sudo: yes
> > +  apt: name=apparmor state=absent
> > +  notify: restart server
>
> As I understand, we also need to add a local action to wait for server to
> be back. Let me find an example.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/DoSomething/tower/pull/9/files#r30063233>.
>